src/backend/tcop/zenith_wal_redo.c

```diff
@@ -608,20 +590,15 @@ ApplyRecord(StringInfo input_message)
 			 record->xl_tot_len, (int) sizeof(XLogRecord) + nleft);
 
 	/* FIXME: use XLogReaderAllocate() */
```

Comment:
The FIXME is obsolete now. (Or should be moved up to where the 'reader_state` is initialized)

Reply:
Remove